scripts/verify_validator_submissions.py

```diff
@@ -153,183 +134,128 @@ class VoteAccount(NamedTuple):
-            print_ok('Vote account has votes.')
-        else:
-            print_warn('Vote account has not voted yet.')
-
-        if vote_account.commission == 100:
-            print_ok('Vote account commission is 100%.')
-        else:
-            print_error('Vote account commission is not 100%.')
-
-        validator_info = validators_by_identity.get(
-            vote_account.validator_identity_address
-        )
-        if validator_info is None:
-            print_error('Validator identity does not exist.')
-            return
-        else:
-            print_ok('Validator identity exists.')
-
-        if validator_info.keybase_username == self.keybase_username:
-            print_ok('Keybase username in form matches username in identity account.')
-        else:
-            print_error('Keybase username in identity account does not match the form.')
-
-        if validator_info.name is not None and validator_info.name.startswith(
-            'Lido / '
-        ):
-            print_ok('Validator identity name starts with "Lido / ".')
-        else:
-            print_error('Validator identity name does not start with "Lido / ".')
-
-        if validator_info.name == self.identity_name:
-            print_ok('Name in identity account matches name in form.')
-        else:
-            print_error('Name in identity account does not mach name in form.')
-
-        if check_keybase_has_identity_address(
-            self.keybase_username, vote_account.validator_identity_address
-        ):
-            print_ok(
-                f'Validator identity public key is on Keybase under {self.keybase_username}.'
-            )
-        else:
-            print_error('Could not verify validator identity through Keybase.')
-
-        token_account = get_token_account(self.st_sol_account_address)
-        if token_account is not None:
-            print_ok('Fee account exists.')
-        else:
-            print_error(f'Fee account {self.st_sol_account_address} does not exist.')
-            return
-
-        if token_account.state == "initialized":
-            print_ok('Token account is in an initialized state.')
-        else:
-            print_error('Token account is not initialized state.')
-
-        if token_account.mint_address == ST_SOL_MINT:
-            print_ok('Fee account is an stSOL account (it has the correct mint).')
-        else:
-            print_error('Fee account is not an stSOL account, the mint is wrong.')
-
-        name = vote_accounts.setdefault(self.vote_account_address, self.validator_name)
-        if name != self.validator_name:
-            print_error(f'Vote account is already in use by {name}.')
-        else:
-            print_ok('Vote account address is unique among responses seen so far.')
-
-        name = identity_accounts.setdefault(
-            vote_account.validator_identity_address, self.validator_name
+def get_vote_account(self: ValidatorResponse) -> Optional[VoteAccount]:
```

Comment:
This diff looks scarier than it is by default. I recommend clicking the cog icon at the top of the review window, and turning on “hide whitespace changes”.